cmd/bundle/summary.go

```diff
@@ -30,30 +31,28 @@ Useful after deployment to see what was created and where to find it.`,
 
 	var forcePull bool
 	var includeLocations bool
+	var resolveConfig bool
 	cmd.Flags().BoolVar(&forcePull, "force-pull", false, "Skip local cache and load the state from the remote workspace")
 	cmd.Flags().BoolVar(&includeLocations, "include-locations", false, "Include location information in the output")
 	cmd.Flags().MarkHidden("include-locations")
+	cmd.Flags().BoolVar(&resolveConfig, "resolve-config", false, "Resolve and output the bundle config")
```

Comment:
suggestion:
```suggestion
	cmd.Flags().BoolVar(&resolveConfig, "show-full-config", false, "Resolve and output the bundle config")
```

Resolve config implies resolution is taking place which is only partially true here. Includes are resolved, but target overrides and variables are not.